lib/Auth/OAuth.php

```diff
@@ -1057,7 +1058,8 @@ private function generateNonce($bits = 64)
         $accumulatedBits = 0;
         $random          = mt_getrandmax();
 
-        for ($totalBits = 0; $random != 0; $random >>= 1) {
+        $totalBits = 0;
+        for ( ; $random != 0; $random >>= 1) {
```

Comment:
This doesn't seem right. Although it may work, the previous version does the exact same thing in one line and use the for loop syntax as it was originally designed. Or am I wrong?